geemap/toolbar.py

```diff
@@ -27,211 +31,130 @@
 
 
 @map_widgets.Theme.apply
-class Toolbar(widgets.VBox):
-    """A toolbar that can be added to the map."""
-
-    @dataclass
-    class Item:
-        """A representation of an item in the toolbar.
+class ToolbarItem(anywidget.AnyWidget):
+    """A toolbar item widget for geemap."""
+
+    _esm = pathlib.Path(__file__).parent / "static" / "toolbar_item.js"
+    active = traitlets.Bool(False).tag(sync=True)
+    icon = traitlets.Unicode("").tag(sync=True)
+    tooltip_ = traitlets.Unicode("").tag(sync=True)
```

Comment:
What's with the trailing `_` on `tooltip`?

Reply:
Unfortunately, "tooltip" is a property already defined on ipywidgets, so it ends up triggering this callback:

https://github.com/jupyter-widgets/ipywidgets/blob/303cae4dc268640a01ce08bf6e22da6c5cd201e4/packages/base/src/widget.ts#L1035

I realized that tooltip_ is a little cryptic and easy to mess up (which I did a few times already), so I changed it to `tooltip_text`.